### DISCUSSION TITLE
I'm pushing my Unreal Project, and got error 500.

### DISCUSSION THREAD

**Discussion by PPYukito** [community/community#79235] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I was pushing my Unreal game project for a really long time and I think there're problems with it so I cancelled the pushing progress and then I pushed it again, and I got error 500.
I searched for it and it said to wait a bit, so I waited and pushed again, still got the same error, can someone help me? Thanks.

**Comment by PPYukito** [comment]
I got it, the files are too large so I have to reset the commit and then separate the files to many commits and push it many times. I mean I committed them to 43 commits, no big file at all.
for those who're having the same problem as me.

"git reset HEAD~" would help you reset your commit.

so in case you get to import big assets to your project again, separate your commit and don't do it at the end of the day or you will get BIG troubles

**Comment by a7medelbatal** [comment]
i don't understand you..

**Reply by PPYukito** [reply]
My bad, allow me to explain this.

I'm studying Unreal Engine 4 and put it on github.
Then I downloaded the model assets to the project and tried pushing it to github but it took too long and didn't finish so I cancelled it and tried pushing it again after I waited for about an hour but I got error 500. I tried searching for error problem and I found one said to wait for it and tried pushing it again, still got error 500.

So I tried another way, I tried reset the latest git commit I did, and now I separated the files and committed them many times instead and it worked just fine.

hope this answer you, sorry if my English is good or my grammar is wrong.
